metacat-main/src/main/java/com/netflix/metacat/main/services/search/ElasticSearchUtilImpl.java

```diff
@@ -371,7 +355,7 @@ private ObjectNode toJsonObject(final ElasticSearchDoc elasticSearchDoc) {
         final QueryBuilder queryBuilder = QueryBuilders.boolQuery()
             .must(QueryBuilders.termsQuery("name.qualifiedName.tree", names))
             .must(QueryBuilders.termQuery("deleted_", false))
-            .must(QueryBuilders.rangeQuery("_timestamp").lte(marker.toDate()))
+            .must(QueryBuilders.rangeQuery(ElasticSearchDoc.Field.TIMESTAMP).lte(marker.toDate()))
```

Comment:
Does this work? We are storing the timestamp in epoch millis.

Reply:
I will update to milli for comparison.

Reply:
Please add a test case